src/app/ruler/ruler.js

```diff
@@ -146,5 +153,18 @@ document.querySelector('.ruler__inner__theme').addEventListener('click', () => {
 window.addEventListener('resize', updateMesures);
 
 ipc.on('settings-changed', loadSettings);
+ipc.on('toggle-center-guides', toggleCenterGuides);
+
+function toggleCenterGuides() {
+	showCenterGuides = !showCenterGuides;
+
+	if (showCenterGuides === true) {
```

Comment:
"truthy" is enough, I just wanted to make sure that the intention of the code was super readable while I was working the logic out in my head, I will update statements like this to be more concise (truthy).